### PR TITLE
Updating Initial Z Parameter Sweep Results

### PR DESCRIPTION
**Pull request by gwaygenomics**
After updates to data processing (#9) and a reorganization of the module (#11), I reran the initial Z sweep. The updated results are below. 

While there are many files updated in this PR (2,837), the primary updates are found in the updated `README.md`. The only script updated is `1.initial-z-sweep/scripts/param_sweep_latent_space_viz.R`.

I also removed the `colorblindr` package dependency because it is not a recipe in conda (see #13).

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
I believe the issue you reference for `colorblindr` should be #14.

File: 1.initial-z-sweep/README.md [hunk 1]
```diff
@@ -98,45 +117,70 @@ Training of models with optimal hyperparameters are shown in **figure 3**.
...
 
-**Figure 5.** The loss of validation sets at the end of training for 720 ADAGE models.
+**Figure 5.** The loss of validation sets at the end of training for 720 untied weight ADAGE models.
 
 A similar pattern appears where lower dimensionality benefits from increased sparsity.
 ADAGE models are also generally stable, particularly at high dimensions.
 
+It appears that `learning rate` is globally optimal at 0.0005; epochs at 100; batch size at 50; sparsity at 0; with decreasing noise for larger z dimensions.
+
+![](figures/z_param_adage/z_parameter_adage_bes.png?raw=true)
```
Comment (jaclyn-taroni) [inline 1.initial-z-sweep/README.md]: This figure does not appear to be showing up here: https://github.com/gwaygenomics/interpret-compression/blob/c6bc2e39f80ecdc62078d4c92f77e2b8b86d583a/1.initial-z-sweep/README.md

File: 1.initial-z-sweep/README.md [hunk 2]
```diff
@@ -98,45 +117,70 @@ Training of models with optimal hyperparameters are shown in **figure 3**.
...
+
+**Figure 6.** Training optimal untied weight ADAGE models across different latent space dimensions.
+
+### Tied Weights
+
+By constrianing the compression and decompression networks to contain the same weights (tied weights), ADAGE models had variable performance across models.
```
Comment (jaclyn-taroni) [inline 1.initial-z-sweep/README.md]: `constraining`

File: 1.initial-z-sweep/README.md [hunk 3]
```diff
@@ -98,45 +117,70 @@ Training of models with optimal hyperparameters are shown in **figure 3**.
...
+
+**Figure 7.** The loss of validation sets at the end of training for 648 tied weight ADAGE models.
+
+After removing `learning rate = 1e-05` and `learning_rate = 5e-05`, we see a clearer picture (**Figure 8**).
+
+![](figures/z_param_adage_tied_weights/z_param_adage_remove_learningrate_tiedweights.png?raw=true)
+
+**Figure 8.** The loss of validation sets at the end of training for 432 tied weight ADAGE models.
+
+It appears the models perform better without any induced sparsity
```
Comment (jaclyn-taroni) [inline 1.initial-z-sweep/README.md]: Missing a period at the end of this sentence

File: 1.initial-z-sweep/README.md [hunk 4]
```diff
@@ -98,45 +117,70 @@ Training of models with optimal hyperparameters are shown in **figure 3**.
...
-| 25 | 0 | 0.0 | 100 | 50 | 0.0005 | 0.013 |
-| 50 | 0.0 | 0 | 100 | 50 | 0.0005 | 0.011 |
-| 75 | 0 | 0.0 | 100 | 50  | 0.0005 | 0.010 |
-| 100 | 0 | 0.0 | 100 | 50 | 0.0005 | 0.009 |
-| 125 | 0 | 0.0 | 100 | 50 | 0.0005 | 0.008 |
-
-It appears that `learning rate` is globally optimal at 0.0005; epochs at 100; batch size at 50; and noise and sparsity at 0.
+| 5 | 0 | 0.0 | 100 | 50 | 0.0015 | 0.0042 |
+| 25 | 0 | 0.0 | 100 | 50 | 0.0015 | 0.0029 |
+| 50 | 0.0 | 0 | 100 | 50 | 0.0005 | 0.0023 |
```
Comment (jaclyn-taroni) [inline 1.initial-z-sweep/README.md]: Why `0.0` instead of `0` here?

File: 1.initial-z-sweep/README.md (hunk 1 above)
Comment (gwaygenomics) [reply 1.initial-z-sweep/README.md]: ah, yeah, typo. My bad

File: 1.initial-z-sweep/README.md (hunk 2 above)
Comment (gwaygenomics) [reply 1.initial-z-sweep/README.md]: :man_facepalming:

File: 1.initial-z-sweep/README.md (hunk 4 above)
Comment (gwaygenomics) [reply 1.initial-z-sweep/README.md]: a typo - will fix

**Pull request merged**